cypress/integration/dashboard.spec.js

```diff
@@ -6,6 +6,26 @@ describe('Dashboard', function () {
     cy.visit('http://localhost:8080/?dashboard=true')
   })
 
+  it ('Renders cells in a row with the same height', function () {
```

Comment:
@pjanik Thanks for looking through this! I think I addressed all your comments, except for updating `cypress:test` to run the server, which is a good idea. I wanted to tag you on the new grid tests I added based on your suggestions - I learned about some new Cypress functions that make chaining much cleaner! They're all documented here:

https://docs.cypress.io/api/introduction/api.html#